src/xperm_rules.md

```diff
@@ -74,6 +75,26 @@ Conditional Policy Statements
 | ----------------------- | ----------------------- | ----------------------- |
 | No                      | No                      | No                      |
 
+### Extended Permission Evaluation
+
+Extended permission rules are evaluated as follows:
+
+* If no extended permissions are defined, only the resource-level policy is
+  considered.
+
+* If an extended permission rule is defined, the policy is first evaluated
+  according to the high-level resource policy. For example:
```

Comment:
See above regarding "high-level resource policy".  Additionally, I think this could spell out the situation more explicitly.  After the AVC and constraint checks are performed, then the xperm checks will be applied.